subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/corefeature/ExcludeRuleMergingPerformanceTest.groovy

```diff
@@ -27,6 +28,7 @@ class ExcludeRuleMergingPerformanceTest extends AbstractCrossVersionPerformanceT
         runner.minimumVersion = '4.0'
     }
 
+    @Ignore
```

Comment:
@melix I think it's important to add a GH issue when making such a change, and to reference in the commit message (or a comment on the PR). Can you please do so now?

Reply:
You mean the comment of the commit itself is not enough? I planned to re-enable today already as soon as a new snapshot is published.

Reply:
Actually the commit comment was fine. I only read the header and failed to read the entire message. Apologies.